BeerProject/spring-boot-ehcache/cache/file/beerCache_cd91a87fe7406136c1d48ded95a88f77c92bbd36/offheap-disk-store/ehcache-disk-store.meta

```diff
@@ -0,0 +1,4 @@
+#Key and value types
```

Comment:
Jezęli spring sam sobie generuje te foldery to w takim razie trzeba dodać je do gitignora i usunąć z repozytorium